module-3/web-app/src/components/Pages/Projects/Projects.module.scss

```diff
@@ -1,15 +1,39 @@
 .container {
     display: flex;
+    align-items: start;
+    align-content: center;
     margin: 0 auto;
     min-height: 90vh;
+    
+    & > a {
+        width: 265px;
+        margin-right: 2em;
+        text-decoration: none;
+        color: white;
+    }
 }
 
 .project {
-    margin-right: 2em;
-    height: fit-content;
-    border: 2px #fff solid;
+    padding: 10px;
```

Comment:
Same as above, instead of `project` use `project-icon-wrapper` or maybe `project-tile-wrapper`, probably would be more suitable in this case.

Reply:
Renamed